third_party/openthread/platforms/qpg/BUILD.gn

```diff
@@ -19,7 +19,8 @@ import("//build_overrides/qpg_sdk.gni")
 import("${qpg_sdk_build_root}/qpg_sdk.gni")
 
 # QPGxxxx settings and OT glue code
-openthread_qpg_family = "${qpg_target_ic}"
+#openthread_qpg_family = "${qpg_target_ic}"
```

Comment:
delete commented out code instead of commenting it out.

A comment saying  that this may not match the actual chip and why would help

Reply:
Done.